### PR TITLE
Rename Layout to LayoutContainer in boilerplate

### LINKED ISSUE #80: Rename boilerplate "Layout" component to something more descriptive
It took me a good minute or two of searching and finding the `import` in the `app/routes.js` file to understand that the `Layout` component of the boilerplate is the wrapper around all routes.

Renaming it to `Wrapper`, `App` or something to that effect should happen, because it's very unclear from the naming alone what the component does.

### PR DESCRIPTION
**Pull request by mxstbr**
Closes #80

### CODE REVIEW

**Comment by mxstbr** [comment]
Not entirely sure why appveyor fails, any ideas?

**Comment by MoOx** [comment]
Nevermind, see #62.

File: boilerplate/web_modules/Layout/index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import Footer from "Footer"
 
 import styles from "./index.css"
 
-export default class Layout extends Component {
+export default class LayoutWrapper extends Component {
```
Comment (MoOx) [inline boilerplate/web_modules/Layout/index.js]: LayoutWrapper here :p

**Comment by mxstbr** [comment]
@MoOx that should do it

**Pull request closed without merging**